### PR TITLE
rbd: lift snapshot, clone and metrics to GA version

### PR DESCRIPTION
**Pull request by humblec**
We have added enough e2e tests and the below functionality is
also proven as working based on the user reports so far. The
support at CO side is also pretty stable for these features.

* snapshot
* clone
* metrics

This commit lift the support state of above features to GA from next
release onwards.

Signed-off-by: Humble Chirammal <hchiramm@redhat.com>

### CODE REVIEW

File: README.md
Comment (Madhu-1) [inline README.md:96]: @nixpanic if I remember correctly you are working on usage details for block volumes. is it done already?

File: README.md
Comment (Madhu-1) [inline README.md:102]: move `Cephfs Dynamically provision, de-provision File mode RWO/RWX volume to GA`

File: README.md
Comment (humblec) [reply README.md:96]: its done, @Madhu-1 https://github.com/kubernetes/kubernetes/pull/97972 and csi driver expose it too.. there are some more enhancement we can do/proposed, but thats an addon to what meant as metrics in the base implementation

File: README.md
Comment (humblec) [reply README.md:102]: @Madhu-1  added these too

**Review by Madhu-1 — APPROVED** [review]

**Review by yati1998 — APPROVED** [review]
LGTM

**Comment by humblec** [comment]
whats wrong, this is not getting merged even after approvals ? @ceph/ceph-csi-contributors

**Comment by humblec** [comment]
> whats wrong, this is not getting merged even after approvals ? @ceph/ceph-csi-contributors

ah.. it was waiting for me to comment :P

**Pull request merged**
